pkg/kubelet/eviction/helpers.go

```diff
@@ -772,9 +772,9 @@ func debugLogObservations(logPrefix string, observations signalObservations) {
 	}
 	for k, v := range observations {
 		if !v.time.IsZero() {
-			klog.Infof("eviction manager: %v: signal=%v, available: %v, capacity: %v, time: %v", logPrefix, k, v.available, v.capacity, v.time)
+			klog.InfoS("eviction manager, signal, available, capacity, time", "logPrefix", logPrefix, "k", k, "available", v.available, "capacity", v.capacity, "time", v.time)
```

Comment:
```suggestion
			klog.InfoS("eviction manager observations", "afterResourceReclaim", afterResourceReclaim, "signal", k, "available", v.available, "capacity", v.capacity, "time", v.time)
```

`logPrefix` is either `observations` or `observations after resource reclaim`. I'd update `logPrefix` to be a bool `afterResourceReclaim` and set it to true or false rather than formatting in this string.